amp.php

```diff
@@ -278,7 +278,7 @@ function amp_deactivate() {
 		}
 	}
 
-	flush_rewrite_rules();
+	flush_rewrite_rules( false );
```

Comment:
The addition of `false` here is something which should have been here from the start, as there is no need to update `.htaccess`.